src/main/flight/pid.c

```diff
@@ -1241,7 +1247,13 @@ void FAST_CODE pidController(const pidProfile_t *pidProfile, const rollAndPitchT
         }
 #endif
         // calculating the PID sum
-        pidData[axis].Sum = pidData[axis].P + pidData[axis].I + pidData[axis].D + pidData[axis].F;
+        const float pidSum = pidData[axis].P + pidData[axis].I + pidData[axis].D + pidData[axis].F;
```

Comment:
I know this isn't much code change, but in general it can't hurt, so I think we should make this functionality conditional to `USE_INTEGRATED_YAW_CONTROL` or similar.